libs/utils/utils.js

```diff
@@ -1147,7 +1147,7 @@ function decorateMeta() {
   const { origin } = window.location;
   const contents = document.head.querySelectorAll(`[content*=".${SLD}."]`);
   contents.forEach((meta) => {
-    if (meta.getAttribute('property') === 'hlx:proxyUrl') return;
+    if (meta.getAttribute('property') === 'hlx:proxyUrl' || meta.getAttribute('name').endsWith('schedule')) return;
```

Comment:
Are there any other meta names with this issue, other than "schedule"? We might be in a situation where someone else has another schedule referencing a URL and they will be forced to update the meta name. If this is just `<meta name="schedule">` we're interested in, could we:
```suggestion
    if (meta.getAttribute('property') === 'hlx:proxyUrl' || meta.getAttribute('name') === 'schedule') return;
```

Reply:
@overmyheadandbody  The metadata can be like `americas_schedule` or `apac_schedule` according to @3ch023 .

Reply:
yes it can be emea_schedule too :) endswith('schedule') seems good to me

Reply:
Should it possibly be `.endsWith('_schedule')` or make it more explicit with `.endsWith('_promo_schedule')`?

Reply:
I think the answer is NO since the issue case (US homepage) has `schedule`. So it can be just schedule w/o prefix.
<img width="535" alt="image" src="https://github.com/user-attachments/assets/6fd73029-0463-4075-b06f-c859b081e2d0">

Reply:
confirming that .endsWith('_schedule') will not be enough since we also need to ignore 'schedule' itself.
I updated the doc with all possible variations of this metadata: https://milo.adobe.com/docs/authoring/metadata (see end of table)

>We might be in a situation where someone else has another schedule referencing a URL and they will be forced to update the meta name. 

I understand, but as of now I am not able to find such a field (checked code and https://milo.adobe.com/docs/authoring/metadata). + It is rather unlikely that the field containing a 'pure' URL would end with a 'schedule' in the name. 
If we find such a field we could switch to the exact check of `===` instead of endsWith, but it will cost more bytes in code. 

From my side it's ok to leave it as .endsWith('schedule'), so approving the PR.